hibernate5/src/main/java/com/baeldung/hibernate/SchemaMultitenantUtil.java

```diff
@@ -0,0 +1,12 @@
+package com.baeldung.hibernate;
+
+public class SchemaMultitenantUtil extends HibernateMultiTenantUtil{
+    @Override
+    public String tenantUrl(String originalUrl, String tenant, String tenantStrategy) throws UnsupportedTenancyException {
```

Comment:
Did you intentionally leave some of this code commented out? Need a test class for the schema strategy that exercises this code.

Reply:
OK. Made two Seperate Classes.